api/authenticator.py

```diff
@@ -953,13 +940,6 @@ def create_bearer_token(self, provider_name, provider_token):
         )
         return jwt.encode(payload, self.bearer_token_signing_secret, algorithm="HS256")
 
-    def decode_bearer_token_from_header(self, header):
```

Comment:
This is no longer required since the Authorization object separates the header for us